src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
```

Comment:
Well done , you are using JSdocs. 
You can also describe arguments  of function, something like this:
```
/**
 * @param {Object[]} people
 * @param {number} century
 * @return {number} averageAge
 */
```